odata-json-format/15 Delta Payload.md

```diff
@@ -129,130 +128,9 @@ collection.
-entities or [entity references](#EntityReference)
-are added to the collection. [Deleted entities](#DeletedEntity) MAY be specified in a nested delta
-representation to represent entities no longer part of the collection.
-If the deleted entity specifies a `reason` as
-`deleted`, then the entity is both removed from the
-collection and deleted, otherwise it is removed from the collection and
-only deleted if the navigation property is a containment navigation
-property. The array MUST NOT contain [added](#AddedLink) or [deleted
-links](#DeletedLink).
-
-::: example
-Example ##ex: 4.01 delta response customers with expanded orders
-represented inline as a delta
-
-  1. Customer `BOTTM`:
-     1. `ContactName` was changed to `Susan Halvenstern`
-     2. Order 10645 was added
-  2. Customer `ALFKI`:
-     1. Order 10643 was removed
-  3. Customer `ANTON` was deleted
-
-```json
-{
-  "@context": "http://host/service/$metadata#Customers/$delta",
-  "@count": 3,
-  "value": [
-    {
-      "@id": "Customers('BOTTM')",
-      "ContactName": "Susan Halvenstern",
-      "Orders@delta": [
-        {
-          "@id": "Orders(10645)"
-        }
-      ]
-    },
-    {
-      "@id": "Customers('ALFKI')",
-      "Orders@delta": [
-        {
-          "@context": "#Orders/$deletedEntity",
-          "@removed": {
-             "reason": "changed"
-          },
-          "@id": "Orders(10643)"
-        }
-      ]
-    },
-    {
-      "@context": "#Customers/$deletedEntity",
-      "@removed": {
-         "reason": "deleted"
-      },
-      "@id": "Customers('ANTON')"
-    }
-  ],
-  "@deltaLink": "Customers?$expand=Orders&$deltatoken=8015"
-}
-```
-:::
-
-If the expanded navigation property is a full representation of the
-collection, it MUST be represented as an expanded navigation property,
-and its array value represents the full set of entities related
-according to that relationship and satisfying any specified expand
-options. Members of the array MUST be represented as
-[added/changed](#AddedChangedEntity) entities or [entity
-references](#EntityReference) and MUST NOT include added links,
-deleted links, or deleted entities. Any entity not represented in the
-collection has either been removed, deleted, or changed such that it no
-longer satisfies the expand options in the defining query. In any case,
-clients SHOULD NOT receive additional notifications for such removed
-entities.
-
-::: example
-Example ##ex: 4.01 delta response for a single entity with an expanded navigation
-property containing only a partial list of related entities (as
-indicated with a [next link](#ControlInformationnextLinkodatanextLink))
-```json
-{
-  "@context": "http://host/service/$metadata#Customers/$entity/$delta",
-  …
-  "Orders@count": 42,
-  "Orders": [ … ],
-  "Orders@nextLink": "…",
-  …
-  "@deltaLink": "Customers('ALFKI')?$expand=Orders&$deltatoken=9711"
-}
-```
-:::
-
 ## ##subsec Deleted Entity
```

Comment:
Examples for 4.0 in this section also need the `odata.` prefix for control information.